### DISCUSSION TITLE
Increase wallpaper file size allowance

### DISCUSSION THREAD

**Discussion by genoism** [bluecaret/carettab#382] [Ideas and Feature Requests]
Please increase the file size allowance to better accommodate large sized gifs (ie. for the 1920x1080 resolution or higher). Typical gifs for this resolution range between 5-10mb in file size.

It'd also be great if we could specify a folder (or something similar) in which the tabs would randomly cycle between the images/gifs in there.

Would gladly make a donation to have this built :)

**Comment by bluecaret** [comment]
Not going to guarantee anything as I have been taking a break from working on CaretTab for a while, but there are a few things I've been considering adjusting in the extension soon, and providing more storage for wallpapers is one. Again, not promising anything soon or at all, but we will see how the next week or two goes for me regarding other things in my life. Thanks for the suggestions!

**Comment by bluecaret** [comment]
Good news! I just submitted to the web store version 4.6.0 which now allows you to upload any wallpaper size! The update should be live in a few days.

The other suggestion for multiple images is already covered in #378.

**Comment by genoism** [comment]
Amazing, thank you!
